app/api/router.py

```diff
@@ -7,6 +8,8 @@
 from app.services.logger import setup_logger
 from app.api.error_utilities import InputValidationError, ErrorResponse
 from app.api.tool_utilities import load_tool_metadata, execute_tool, finalize_inputs
+from fastapi.responses import FileResponse
```

Comment:
Please, don't modify the `router.py`. Use the conventional one and just return JSON data for keeping consistency.